make/jdk/src/classes/build/tools/cldrconverter/CLDRConverter.java

```diff
@@ -1372,6 +1372,7 @@ private static Map<Locale, String> coverageLevelsMap() throws Exception {
     private static void generateTZDBShortNamesMap() throws IOException {
         Files.walk(Path.of(tzDataDir), 1, FileVisitOption.FOLLOW_LINKS)
             .filter(p -> p.toFile().isFile())
+            .filter(p -> p.getFileName().toString().matches("africa|antarctica|asia|australasia|backward|backzone|etcetera|europe|factory|northamerica|southamerica"))
```

Comment:
I am just curious, why is this needed?

Reply:
It would otherwise scan all files under the directory, and fails with `iso3166.tab` or `zone.tab`, which include "LI" as the country code at the beginning of the line, that interfere with "Link" match.
Having said that, the default list was different from the one in `TzdbZoneRulesProvider`. I removed extra file names.

Reply:
Right, the match in general is much more lenient now, so need to restrict the files searched for false positives. Thanks for the explanation, that makes sense.